builtin/client/sscsm.lua

```diff
@@ -168,23 +179,21 @@ function Env:exec(code, file)
 	return true
 end
 
--- Create the "base" environment
-local base_env = Env:new_empty()
-function Env.new() return base_env:copy() end
+-- Create the environment
+local env = Env:new_empty()
 
 -- Clone everything
-base_env:add_globals("assert", "dump", "dump2", "error", "ipairs", "math",
+env:add_globals("assert", "dump", "dump2", "error", "ipairs", "math",
 	"next", "pairs", "pcall", "select", "setmetatable", "string", "table",
-	"tonumber", "tostring", "type", "vector", "xpcall", "_VERSION", "utf8")
```

Comment:
Are you sure `utf8` isn't needed here anymore?

Reply:
I didn't realise that got removed when I was backporting changes, I've re-added it.